### PR TITLE
add ref links to the README

### PR DESCRIPTION
**Pull request by pjfanning**
* link to pekko docs
* add link to users mailing list
* remove the Software Mill logo - I don't think it is great for ASF projects to have commercial logos
  * we do properly acknowledge Software Mill in the license file and source headers

### CODE REVIEW

**Comment by mdedetrich** [comment]
> remove the Caveat Emptor - we have marked this repo as being stable (semver setting in sbt)

Are we sure this is a wise decision, i.e. should we discuss this on  the mailing list?

**Comment by pjfanning** [comment]
> > remove the Caveat Emptor - we have marked this repo as being stable (semver setting in sbt)
> 
> Are we sure this is a wise decision, i.e. should we discuss this on the mailing list?

* if we are keeping the 'Caveat Emptor', we need to remove the sbt setting for semver
* is this connector not quite stable at this stage?

**Comment by mdedetrich** [comment]
> if we are keeping the 'Caveat Emptor', we need to remove the sbt setting for semver

Yes this is what I meant (sorry if it wasn't implied). I am making a general question about whether we are sure if abiding by semver is the right decision

> is this connector not quite stable at this stage?

I don't know (thats why I want to ask on mailing list). I assume that this message was also in Akka and if they put it there it was for a reason.

**Comment by pjfanning** [comment]
@mdedetrich I added back the caveat

**Comment by mdedetrich** [comment]
I would also remove https://github.com/apache/incubator-pekko-connectors-kafka/blob/main/build.sbt#L19, me adding it in there was a mistake (I didn't realize at the time that this pekko module was marked as having non stable ABI evolution).

File: README.md [hunk 1]
```diff
@@ -19,30 +18,25 @@ Pekko connectors to other technologies are listed in the [Pekko connectors repos
...
-------------
+## Contributing
 
 Contributions are very welcome. If you have an idea on how to improve Pekko, don't hesitate to create an issue or submit a pull request.
 
 See [CONTRIBUTING.md](CONTRIBUTING.md) for details on the development workflow and how to create your pull request.
 
-Caveat Emptor
--------------
+## Caveat Emptor
```
Comment (mdedetrich) [inline README.md:40]: You forgot to add back in the `-------------`

**Review by mdedetrich — APPROVED** [review]
Ill approve, you forgot to revert one change completely.

File: README.md (hunk 1 above)
Comment (pjfanning) [reply README.md:40]: I used `##` header marker instead - I changed all the headers in this PR to use that markdown style - more consistent with out other md files

**Comment by pjfanning** [comment]
> Ill approve, you forgot to revert one change completely (see [#98 (comment)](https://github.com/apache/incubator-pekko-connectors-kafka/pull/98#discussion_r1254776848))

I added another commit to remove the semver setting in sbt file

**Pull request merged**